website/pages/docs/k8s/connect/ingress-gateways.mdx

```diff
@@ -5,4 +5,203 @@ sidebar_title: Ingress Gateways
+
+* Setting the helm chart configuration
+* Deploying the helm chart
+* Configuring the gateway
+* Defining an Intention (if ACLs are enabled)
+* Deploying your application to Kubernetes
+* Connecting to your application
+
+## Setting the helm chart configuration
+When deploying the helm chart you must provide helm with a custom yaml file that contains your environment configuration.
+
+```yaml
+global:
+  name: consul
+connectInject:
+  enabled: true
+ingressGateways:
+  enabled: true
+  gateways:
+    - name: ingress-gateway
+      replicas: 2
+      service:
+       type: LoadBalancer
+```
+~> *Note:* this will create a public unathenticated LoadBalancer in your cluster, please take appropriate security considerations.
+
+The yaml snippet is the launching point for a valid configuration that must be supplied when installing using the [official consul-helm chart](https://hub.helm.sh/charts/hashicorp/consul).
+Information on additional options can be found in the [Helm reference](/docs/k8s/helm). Configuration options for ingress gateways reside under the [ingressGateways](/docs/k8s/helm#v-ingressgateways) entry.
+
+
+The gateways stanza is where you will define and configure the set of ingress gateways you want deployed to your environment.
+The only required field for each entry is `name`, though entries may contain any of the fields found in the `defaults` stanza.
+Values in this section override the values from the defaults stanza for the given ingress gateway with one exception:
+the annotations from the defaults stanza will be *appended* to any user-defined annotations defined in the gateways stanza rather than being overridden.
+Please refer to the ingress gateway configuration [documentation](/docs/k8s/helm#v-ingressgateways-defaults) for a detailed explanation of each option.
+
+-> *Note*: Make sure any ports that will be used as listeners in the ingress gateway's Consul config entry are included in the `ports` object for each gateway.
+
+## Deploying the helm chart
+
+Ensure you have the latest consul-helm chart and install Consul via helm using the following
+[guide](/docs/k8s/installation/overview#installing-consul) while being sure to provide the yaml configuration
+as previously discussed.
+
+## Configuring the gateway
+
+Now that Consul has been installed with ingress gateways enabled, you must add the corresponding configuration to Consul. This requires you to use the Consul CLI.
+Configuring the ingress gateway requires:
+
+* Accessing the Consul server
+* Submitting an Ingress Gateway configuration entry to Consul
+
+### Accessing the Consul server
+
+You can access the Consul server directly from your host via `kubectl port-forward`, this is helpful for interacting with the Consul UI locally as well as validating connectivity of the application.
+
+```shell-session
+$ kubectl port-forward consul-server-0 8500 &
+```
+-> If TLS is enabled use port 8501
+
+-> Download the latest Consul binary from [Downloads](/downloads.html).
```

Comment:
Should this be the binary that matches what they've installed? I'm wondering if a version mismatch would cause any problems.

Reply:
Good point, I think with the combination of the notes at the top of the page regarding version requirements and this link pointing to at least those versions it should be okay, what do you think?